test/parallel/test-cluster-worker-wait-server-close.js

```diff
@@ -22,10 +22,12 @@ if (cluster.isWorker) {
   worker.once('listening', function() {
     net.createConnection(common.PORT, common.localhostIPv4, function() {
       var socket = this;
-      worker.disconnect();
-      setTimeout(function() {
-        socket.write('.');
-        connectionDone = true;
+      setTimeout(function(){
```

Comment:
Please, see this comment: https://github.com/nodejs/io.js/pull/1400#issuecomment-110472005